pkg/sdk/testint/grants_integration_test.go

```diff
@@ -842,10 +842,10 @@ func TestInt_GrantAndRevokePrivilegesToDatabaseRole(t *testing.T) {
 func TestInt_GrantPrivilegeToShare(t *testing.T) {
 	client := testClient(t)
 	ctx := testContext(t)
-	shareTest, shareCleanup := testClientHelper().Share.CreateShare(t)
+	shareTest, shareCleanup := testClientHelper().Share.CreateShareWithIdentifier(t, testClientHelper().Ids.RandomAccountObjectIdentifierContaining(".foo.bar"))
```

Comment:
it would be better to have a separate test to check this (one may wonder why this name is so strange - the dedicated test would explain this)

Reply:
Done.